config/crd/kustomization.yaml

```diff
@@ -4,3 +4,4 @@ kind: Kustomization
 resources:
 - bases/postgres-operator.crunchydata.com_postgresclusters.yaml
 - bases/postgres-operator.crunchydata.com_pgupgrades.yaml
+- bases/postgres-operator.crunchydata.com_pgadmins.yaml
```

Comment:
I know we talked about this a bit yesterday, but I wonder if a kustomize overlay might solve the "problem" of having a regular deploy (without crd, with old rbac) and a standalone_pgadmin deploy? (Similar to the way we have our default, cluster-wide deploy and our single-namespace deploy.)

(Also, I say "problem" because I don't really think having it in is a problem.)

Reply:
That's a good idea. I'll plan to look into that as an option.

Reply:
Looked into this a bit more and, while it definitely looks possible, it's a little heavier of a lift than I'd like for this initial effort. Assuming everyone's onboard, I'd like to leave as is for now.